README.md

```diff
@@ -10,35 +10,38 @@ Creating what we call [starter issues](http://hood.ie/blog/starter-issues.html)
 
 ### 💡💥❓ How things work
 
-Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request, I create a new branch that is called something like first-timers-only-typo-in-title. GitHub will notify the "first-timers-only issue bot" about the new branch using Webhooks. Currently, the bot will see that a new branch was created starting with "first-timers-only" and it will create a new issue at https://github.com/hoodiehq/camp/issues/new based on a template and assign the first-timers-only and the up-for-grabs label. If the commit body can be used to add some context information. If left empty, the 🤔 What you will need to know section of the issue will simply say "Nothing :)". Otherwise the commit body will be used there.
+Say I’m a Hoodie contributor and find a typo somewhere. Instead of fixing the issue directly in the master branch or creating a pull request which is time-consuming, I can simply create a new branch that is called something like **first-timers-only-typo-in-title.** GitHub will then notify the "first-timers-only issue bot" about the new branch using Webhooks. The bot is listening to any new branch starting with **first-timers-only** and it will create a new issue on your repo. It is currently setup to the Hoodie repo with a template and assigns the first-timers-only and up-for-grabs labels but can be modified by the developer. The commit body can be used to add some context information and if left empty, the 🤔 **What you will need to know** section of the issue will simply say "Nothing :)".
+
+
+### 😮 🙌 👀 🎉 Use Our Bot!
+
+<table>
+    <tr>
+        <th>Steps</th>
+        <th>Example</th>
+    </tr>
+    <tr>
+        <td>1) <a href="https://github.com/apps/first-timers-bot">Install App</a> on a repo of your choice</td>
```

Comment:
and an empty `<td></td>`. Or add a screenshot of the install page?

![image](https://user-images.githubusercontent.com/39992/29905904-2815491e-8dc6-11e7-8a94-8c82815da5f5.png)